CHANGES

```diff
@@ -1,6 +1,8 @@
 v3.0.4 - YYYY-MMM-DD (to be released)
 -------------------------------------
 
+ - Adds support to multiple ranges in ctl:ruleRemoveTargetById
+   [Issue #2110 - @theMiddleBlue, @zimmerle]
```

Comment:
@j0k2r why you did put my name on it and where is yours? :)

Reply:
Oh, sorry I don't know the rules, I took the the issue "participants"